homeassistant/components/duckdns/config_flow.py

```diff
@@ -79,3 +81,37 @@ async def async_step_import(self, import_info: dict[str, Any]) -> ConfigFlowResu
 
         deprecate_yaml_issue(self.hass, import_success=True)
         return result
+
+    async def async_step_reconfigure(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Handle reconfigure flow."""
```

Comment:
It feels strange to have a reconfigure flow and no reauth flow.
Is it not possible to detect an invalid access token, and trigger a reauth flow?

We generally have both:
- reauth when the system detects outdated credentials
- reconfigure when the user wants to change credentials (may also want to change other settings but that wouldn't apply to Duck DNS)